homework-submission/homework-week-2/react.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-undef */
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/jsx-no-undef */
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.addTodo = this.addTodo.bind(this);
+  }
+
+  state = {
+    todos: [
+      {
+        id: 1,
+        description: 'Get out of bed',
+        deadline: '2017-09-11',
+        done: true
+      },
+      {
+        id: 2,
+        description: 'Brush teeth',
+        deadline: '2017-09-10',
+        done: false
+      },
+      {
+        id: 3,
+        description: 'Eat breakfast',
+        deadline: '2017-09-09',
+        done: false
+      }
+    ]
+  };
+
+  addTodo(e) {
```

Comment:
`addTodo = e =>` will remove the need to bind

Reply:
Has been changed.